plugins/slack/alerta_slack.py

```diff
@@ -40,6 +40,8 @@ def post_receive(self, alert):
             color = "#FFFF00"  # yellow
         elif alert.severity == 'warning':
             color = "#1E90FF"  # blue
+        elif alert.severity == 'information':
```

Comment:
This should be `informational` not `information` see https://github.com/guardian/alerta/blob/master/alerta/settings.py#L65